### PR TITLE
Add Support info page and couple of other amendments

### LINKED ISSUE #852: Extra troubleshooting info
1) How to set log level to debug
2) How to find out versions of products
3) Where to find logs
4) Profiling gateway

To make support a bit easier :)

### PR DESCRIPTION
**Pull request by marksou**
Fixes #852

### CODE REVIEW

File: tyk-docs/content/troubleshooting/tyk-gateway/support-information.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Support Information
+menu:
+  main:
+    parent: "Tyk Gateway"
+weight: 8
+url: "/troubleshooting/tyk-gateway/support-information"
+---
+
+When contacting support, you may be asked to supply extra information and supply log files, etc, so we can quickly handle your issue. Questions may include:
```
Comment (christtyk) [inline tyk-docs/content/troubleshooting/tyk-gateway/support-information.md]: Can we change 'issue' to 'request' please?

Comment (marksou) [reply tyk-docs/content/troubleshooting/tyk-gateway/support-information.md]: Will do

**Review by christtyk — APPROVED** [review]
Just that one note from me, Leon for the technical details

File: tyk-docs/content/troubleshooting/tyk-gateway/support-information.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### How do I change my Gateway Logging Level?
+
+You can set the logging level in your `tyk.conf` by adding the following:
+
+```{.copyWrapper}
+  "log_level": "info",
+```
+
+By default, the setting is `info`. You also have the following options:
```
Comment (buger) [inline tyk-docs/content/troubleshooting/tyk-gateway/support-information.md:55]: Pls add a note that it also can be set via `TYK_LOGLEVEL` environment variable.

**Comment by buger** [comment]
Note that we need such info for our whole stack. However, for the rest of components, it is limited by setting debugging level by env variable like: `TYK_LOGLEVEL=debug`

File: tyk-docs/content/troubleshooting/tyk-gateway/support-information.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/troubleshooting/tyk-gateway/support-information.md:55]: Have now added that.

**Comment by buger** [comment]
Environment variable should be `TYK_LOGLEVEL`.

Also `How do I change my Gateway Logging Level?` can be renamed to more general like:
> How do I increase log verbosity?
And then state that for ALL products not only GW, you can set it via env variable and for GW it can be set via config file as well.

**Comment by buger** [comment]
`TYK_GW_TYK_LOGLEVEL` should be renamed to TYK_LOGLEVEL

**Comment by christtyk** [comment]
@marksou is there an update on this one?

**Comment by marksou** [comment]
@buger Can you review again after latest updates?

**Comment by christtyk** [comment]
@buger are you ok to review?

**Comment by adelowo** [comment]
Fat fingers. Apologies :)

**Comment by christtyk** [comment]
bump

**Comment by christtyk** [comment]
@buger can you review?

**Comment by christtyk** [comment]
@marksou 
can you check this page to make sure the content is still in date for me?

**Pull request merged**